### PR TITLE
Volume19

### PR DESCRIPTION
**Pull request by selmaelba**

### CODE REVIEW

File: diary-volumes/volume19.xml [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+                <dateline>
+                    <date when="1913-04-07">Monday. April 7th.</date>
+                </dateline>
+                <p><persName ref="#Rathbone_Elena">Elena Rathbone</persName> and her friend <persName ref="#Cholanley_Miss">Miss Cholanley</persName> just back from <placeName ref="#India">India</placeName> came to tea, as well as <persName ref="#Anthony_Mr">Mr. Anthony</persName> and <persName ref="#Lythgoe_Albert">Mr. Lythgoe</persName>.</p>
+            </div>
+            <div xml:id="EBA-1913-04-12" type="entry">
+                <dateline>
+                    <date when="1913-04-12">Saturday. April 12th.</date>
+                </dateline>
+                <p>Arrived in <placeName ref="#Naples">Naples</placeName> at 7:30 this morning, landed in the hideous confusion characteristic of the N.G.L. steamers <!--The North German Lloyd Line tag?-->- rain and cold winds.</p>
```
Comment (selmaelba) [inline diary-volumes/volume19.xml:199]: Yes, this is a North German Line Steamer

File: diary-volumes/volume19.xml [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
...d <persName ref="#Crispell_Mrs">Mrs. Crispell</persName> and her niece <!--persName tag?--> came from the <name type="boat" ref="#Mayflower">Mayflower</name> <!--boat?-->to tea - I was so pleased with her. <persName ref="#Crane_Lancelot">Mr. Crane</persName> and <persName ref="#Burton_Harry">Harry</persName> came to tea, and we left <placeName ref="#Luxor">Luxor</placeName> the <date when="1913-03-12">12th</date> at noon - having had a nice visit from <persName ref="#Mohassib_Mohammed">Old Hagg Mahassib</persName> - <placeName ref="#Naj_Hammadi">Nag Hamadi</placeName> the <date when="1913-03-13">13</date> - <placeName ref="#Balyana_Al">Balliana</placeName> the <date when="1913-03-14">14</date> - where we stopped for <persName ref="#Newberry_Mary">Mary</persName>, <persName ref="#Newberry_Doris">Doris</persName>, <persName ref="#Amelie">Amelie</persName>, <persName ref="#Jones_Daniel">Jones</persName> and <persName ref="#Hassein">Hassein</persName> to visit <placeName ref="#Abydos">Abydos</placeName> - very cold and terrific wind - very severe in the night and rough today. We passed the <placeName ref="#Arabia_Peninsula">Arabia</placeName> stuck at 8:30 A.M. Cold, constant wind.</p>
```
Comment (selmaelba) [inline diary-volumes/volume19.xml]: I'm not sure that we'll be able to find any information on the niece-- you may as well leave it untagged for now. The Mayflower is a boat, but not the famous one

File: diary-volumes/volume19.xml [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+            </div>
+            <div xml:id="EBA-1913-03-02" type="entry">
+                <dateline>
+                    <date when="1913-03-02">Sunday. March 2nd.</date>
+                </dateline>
+                <p>On <date when="1913-02-24">Monday</date> <persName ref="#Newberry_Mary">Mary</persName> and I went to tea with the <persName ref="#Contardone_Contessa">Contessa</persName> in the gardens of the <name type="hotel" ref="#Luxor_Hotel">Luxor Hotel</name> - met <persName ref="#Baring_Lady">Lady Baring</persName>, <persName ref="#Murray_Miss">Miss Murray</persName>, and <persName ref="#Pappenheim_Contessa">Contessa Pappenheim</persName> and daughter <!--tag daughter?-->. Charming people all - yesterday we left for <placeName ref="#Aswan">Asw&#226;n</placeName>.</p>
```
Comment (selmaelba) [inline diary-volumes/volume19.xml:162]: I don't think it'll be possible to find information on the daughter at this time

File: diary-volumes/volume19.xml [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+            </div>
+            <div xml:id="EBA-1913-02-23" type="entry">
+                <dateline>
+                    <date when="1913-02-23">Sunday. Feb. 23rd.</date>
+                </dateline>
+                <p>Very warm. <persName ref="#Whymper_Charles">Mr. Whymper</persName> to breakfast - the <persName ref="#Pier_Mr #Pier_Mrs">Piers</persName> to lunch. <persName ref="#Aylener_Mrs">Mrs. Aylener</persName> and <persName ref="#Morehouse_Mrs">Morehouse</persName> <!--Mr. or Mrs.?-->to tea. We have had <persName ref="#Hardy_Mr #Hardy_Mrs">Mr. and Mrs. Hardy</persName> to tea - <persName ref="#Nelson_Aldrich">Senator Aldrich</persName> also. <persName ref="#Newberry_Mary">Mary N.</persName> and I have been over to the <name type="hotel" ref="#Winter_Palace">Winter Palace</name> for tea - we looked very smart on Washington's birthday with all our flags.</p>
```
Comment (selmaelba) [inline diary-volumes/volume19.xml]: Likely Mr. Morehouse

**Review by emmafritzberg — APPROVED** [review]

**Pull request merged**